### ISSUE TITLE
PersistentWindows fails to activate in Task Scheduler

### ISSUE THREAD

**Issue by Thisisaaccountyes** [kangyu-california/PersistentWindows#210]
As instructed, after downloading PW, I set up a task in Task Scheduler so it would turn on whenever I logged in. I've used Task Scheduler before and just now ran into this problem. 

I would always get Error result 0x40010004. No matter how much I tried solutions, they never changed. Running with highest privileges, no running with highest privileges, giving PW itself the ability to run as admin, updating to 5.40 (After downgrading myself on purpose with an old installer), downloading 5.40, completely wiping PW from my pc and then reinstalling (Doing both an older version and 5.40), checking for error message 9999 and 9990 (none exist since PW doesn't even activate through Task Scheduler.) 

There's likely more I did, but I'll be rambling on. I've been at this problem for half a month and am not quite sure if there's a fix. If it means anything, I recently upgraded my pc to a whole new system, swapping my storage from my old pc to the new one. 

(Note: I can manually run PW from the folder I shoved it in or in Task Scheduler by going to my custom task and selecting run. That's the life I've resigned myself to ever since. Also I apologize if I summited a bug error wrong, first time ever doing this.) 

Edit: Forgot to mention I am on Windows 10

**Comment by kangyu-california** [comment]
Make sure to  select "Run only when user is logged on" in task property setting.
Also please try "-delay_start 10" command option as instructed in readme.

If none of those works, please make a screen copy of your task settings.

**Comment by Thisisaaccountyes** [comment]
![image](https://user-images.githubusercontent.com/125930805/220219250-7a3b3585-5b4c-4619-851b-528c5e72919e.png)

![image](https://user-images.githubusercontent.com/125930805/220219933-06f0b396-58a5-4ccd-adba-0599ec01544b.png)

![image](https://user-images.githubusercontent.com/125930805/220220314-ce6edbe4-8047-4bbe-a106-80bd62f1b1c5.png)

I have tried using that command before, still am using it now. It has not worked (unless I have been putting it in the wrong text box.)

**Comment by kangyu-california** [comment]
Please upload the "Triggers" screen shot.
Maybe you need to create a trigger to start the PW task when user log on.
The trigger setting may seem redundant to that in "General" tab, nobody knows all the tricks in Microsoft. product.

My thought is that your issue is not PW specific. You might want to raise the 0x40010004 issue to broader audience for better support.

**Comment by Thisisaaccountyes** [comment]
I have tried writing a trigger before, tried several different variations. As of now, there is none. Could you perhaps tell me a specific trigger to set up or if you think that the trigger may not be a fix to the problem, what forum should I ask?

**Comment by kangyu-california** [comment]
![image](https://user-images.githubusercontent.com/59128756/220233725-e7a70188-81a0-4071-9264-9c658f43022b.png)

I don't have other clue right now, sorry about that.

**Comment by Thisisaaccountyes** [comment]
Well that is unfortunate, for now I will try to find a answer and post it here if I do come up positive. Thank you for your time.

**Comment by kangyu-california** [comment]
https://answers.microsoft.com/en-us/windows/forum/all/task-scheduler-last-run-result-code-0x40010004/bc0a35ec-b6f1-4652-919a-ff3a9d54795d

**Comment by Thisisaaccountyes** [comment]
Update! (btw I read that link a long time ago, but sadly wasn't too much help.)

So this applies to 5.40 and current (as of posting) 5.41. This potentially might be even more useful than the current method of setting up a Task Scheduler. The fix for this problem is simple. 

Windows + R (To start the Run program)
Type in "shell: startup" or use file explorer to find the startup folder if you know how to. 
Find your folder or wherever you placed PersistentWindows
Create a shortcut by right-clicking PersistentWindows application and finding the 'create shortcut' option. 
Drag the shortcut into the folder, and done! On startup, it just works! 

**Notes**: Remove admin access on PersistentWindows in properties; the admin permission popup prompt will not allow it to start up on its own automatically.

Make sure there is no more script in Task Scheduler for PersistentWindows. Delete it!

If you don't want to use Run for whatever reason, the direct link for it in File Explorer is: 
C:\Users\[Username]\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup

This is in advance for the creator of this fantastic program. I am new to GitHub and need to learn how things work here. I apologize if I did something wrong or annoying to grab your attention to this closed issue. If I did, I apologize for pestering you into seeing this in advance, and it would be fantastic if you taught me my mistakes and what is the traditional way it is done in GitHub. I would like your feedback if this alternative method was used before or mentioned to you. What are the drawbacks/cons of utilizing this method of automatic startup?
